sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -661,16 +661,16 @@ class AstBuilder extends SqlBaseParserBaseVisitor[AnyRef] with SQLConfHelper wit
   }
 
   override def visitNamedExpressionSeq(
-      ctx: NamedExpressionSeqContext): Seq[Expression] = {
+      ctx: NamedExpressionSeqContext): Seq[(Expression, String)] = {
```

Comment:
this is visiting named expression, do we still need to return an alias?

Reply:
@cloud-fan This function handles not only named expressions, try to run even the simple example:
```sql
select 1
```